docs/sdk/client-side-sdks/react-native/react-native.md

```diff
@@ -31,3 +31,25 @@ The SDK is available as a package on npm. It is also open source and can be view
 ## Requirements: 
 
 This SDK is compatible with _React Native_ version 0.64.0 and above.
+
+## Expo + React Native Web
+
+To get your Expo or React Native Web working with DevCycle, you will need to change one of the rules in the webpack config to exclude `.cjs` files from being transpiled, e.g.:
```

Comment:
we aren't excluding `.cjs` from being transpiled, we are preventing them from being incorrectly treated as asset resources rather than source files. The problem is that they _werent_ being bundled / transpiled. Would also probably be good to link the same issue here that we have in the React docs.